### PR TITLE
pinging configurable and simplified, child exit improved

### PR DESCRIPTION
**Pull request by rupnikj**

### CODE REVIEW

File: src/distributed/topology_local_wrapper.ts [hunk 1]
```diff
@@ -11,11 +11,10 @@ import * as serialize_error from "serialize-error"
 export class TopologyLocalWrapper {
 
     private uuid: string;
-    private process: NodeJS.Process;
+    private process: any; // NodeJS.Process
```
Comment (bergloman) [inline src/distributed/topology_local_wrapper.ts]: write longer comment, explain that methods are missing

File: src/distributed/topology_local_wrapper.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import * as topology_compiler from "../topology_compiler";
-import * as tl from "../topology_local";
-import * as intf from "../topology_interfaces";
-import * as log from "../util/logger";
-import * as serialize_error from "serialize-error"
+import * as topology_compiler from '../topology_compiler';
```
Comment (bergloman) [inline src/distributed/topology_local_wrapper.ts]: Why single quotes?

File: src/distributed/topology_local_wrapper.ts (hunk 1 above)
Comment (rupnikj) [reply src/distributed/topology_local_wrapper.ts]: fixed

File: src/distributed/topology_local_wrapper.ts (hunk 2 above)
Comment (rupnikj) [reply src/distributed/topology_local_wrapper.ts]: fixed

**Pull request merged**